docs/edit/manifest.md

```diff
@@ -0,0 +1,130 @@
+### Context
+
+The usual way to declare what will be tested or not are decorators (`@released`, `@bug`, `@missing_feature`...). This solution offers several advantages :
+
+- declarations are as close as possible to the test, making the link obvious
+- and complex condition (several components involved, complex version range...) is easy to do, as it's declared as python code
+
+### Issue
+
+Unfortunatly, it comes with a major drawback: Activating a single test can become a hell due to confilct between PRs.
+
+### Proposition
+
+To solve this, we'll try to leverage two points :
+
+1. Often, those declarations involve only one component
+2. Often, a component is modified by only one developper
+
+The idea is to offer an alternative way to declare those metadata, using one file per component, also known as manifest file. Those files will be inside system tests repo, declaring for a given component some metadata.
+
+Pro :
+
+- Will solve the majority of conflict during test activation PRs
+- each team will have ownership of its manifest file, easing the PR preview process
+
+Cost :
+
+- declarations will be a little bit far away from the test, asking a small addition in the learning curve
+- re-naming will be a little bit harder, and will spam all teams
+
+## Implementation
+
+- Manifests files will be inside `utils/manifest/`, one per term, named with the argument name used in decorators (`utils/manifest/golang.yml`, `utils/manifest/agent.yml`...)
+- Each team will have ownership of its manifest file
+- pytest will load those files, and decorate tests node during collection
+
+## Supported features
+
+Will suport:
+
+- declaring a skip reason (bug, flaky, irrelevant, missing_feature) for entire file
+- declaring released version (or a skip reason) for a class
+- declaring released version (or a skip reason) for a class, with details for weblog variants
+- The wildcard `*` is supported for weblog declarations
+
+Won't support:
```

Comment:
Are the complex syntaxes like `rules_version > xxx`, `waf_version < xxx && lib_version == xxx` or `xxx < lib_version < yyy` supported ? the supported/not supported lists should precise a bit more on those cases i feel like ?

Reply:
* `rules_version > xxx` -> yes, inside the manifest file dedicated to rules file
* `waf_version < xxx && lib_version == xxx` -> nope. Complex conditions will remain in the code (there is no limit to this complexity, so we need to draw a line based on  the ratio format simplicity / number of occurrences : supporting those use cases will complexify **a lot** the manifest format - it does not happen a lot, and mostly at method level which won't be supported.
* `xxx < lib_version < yyy` -> nope either, same reason

Reply:
- Oh I didn't realize there would be a manifest for the rules, was that in the document or should we add that precision ?
- I'm fine with more complex conditions to not be supported, but by reading this document I'm not really sure where that cutoff point is, is it possible to be more precise ?

Reply:
> Oh I didn't realize there would be a manifest for the rules, was that in the document or should we add that precision ?

In a senss, yes, but I made it crystal clear in  https://github.com/DataDog/system-tests/pull/1338/commits/f65626e8cb884e28acdda1fd1240490116fe4c9d

> I'm fine with more complex conditions to not be supported, but by reading this document I'm not really sure where that cutoff point is, is it possible to be more precise ?

It's in the "Supported features" section, but I make it more precise here : https://github.com/DataDog/system-tests/pull/1338/commits/ecda023165d266963f42c4b2ffb7087daffc08bf

Reply:
`term` -> `team` ok makes more sense now xD